examples/epithelial_example.gro

```diff
@@ -0,0 +1,84 @@
+// This program creates a layer of gut epithelial cells with some section
+// starting off inflamed, emitting an inflammatory signal.
+// E. Coli begin growing around the gut layer, and will fluoresce if they
+// are near the inflamed region. Try it out:)
+
```

Comment:
You should add some author and date information here, in case someone wants to ask questions later.